README.md

```diff
@@ -75,10 +75,10 @@ Install dependencies for Fedora/CentOS systems:
 yum install make autoconf automake libtool popt-devel xz
 ```
 
-Compilation (`autoreconf` is optional if you obtained source from tarball):
+Compilation (`./autogen.sh` is optional if you obtained source from tarball):
 ```
 cd logrotate-X.Y.Z
-autoreconf -fiv
+./autogen.sh
```

Comment:
My thinking was (and is) that it's best to recommend to use facilities provided by project. One day autogen.sh might contain more commands, lets say logrotate will be translated and gettext stuff has to be taken care, and then it is nice if people are used ot do what is considered the right way of bootstrapping the project.  But right now the argument in behalf of autogen.sh is not very strong, so it's fine to say autoreconf does what is needed.